vendor/github.com/sylabs/sif/pkg/sif/sif.go

```diff
@@ -148,6 +148,7 @@ const (
 	FsExt3                      // EXT3 file system, RDWR (deprecated)
 	FsImmuObj                   // immutable data object archive
 	FsRaw                       // raw data
+	FsEncrypt                   // Encrypted File System
```

Comment:
same

Reply:
Should I make these changes separately in sif repository and push?

Reply:
I would fork the [sif repo](https://github.com/sylabs/sif), then checkout your branch and add these changes, push to your branch and open a PR. _Thats how I would do it..._

Then you can update the vendor in this PR.